OpenSees.tcl

```diff
@@ -51,6 +51,7 @@ proc OpenSees::InitGIDProject { dir } {
 	variable OpenSeesEXE; # OpenSees executable with path
 
 	set OpenSeesProblemTypePath $dir
+	array set problemtype_local [GidUtils::ReadProblemtypeXml [file join $dir OpenSees.xml] Infoproblemtype {Name Version MinimumGiDVersion}]

```

Comment:
Needs to be done in order to load the icon of the problemtype in Data->ProblemTypes